internal/test/test.go

```diff
@@ -55,43 +59,60 @@ func run(
 	_ command.GlobalFlags,
 	services *services.Services,
 ) (command.Result, error) {
+	testFiles := make(map[string][]byte, 0)
+	for _, filename := range args {
+		code, err := readerWriter.ReadFile(filename)
 
-	filename := args[0]
+		if err != nil {
+			return nil, fmt.Errorf("error loading script file: %w", err)
+		}
 
-	code, err := readerWriter.ReadFile(filename)
-	if err != nil {
-		return nil, fmt.Errorf("error loading script file: %w", err)
+		testFiles[filename] = code
+	}
+
+	var coverageReport *runtime.CoverageReport
+	if testFlags.Cover {
+		coverageReport = runtime.NewCoverageReport()
 	}
 
 	result, err := services.Tests.Execute(
-		code,
-		filename,
+		testFiles,
 		readerWriter,
+		coverageReport,
 	)
 
 	if err != nil {
 		return nil, err
 	}
 
+	if coverageReport != nil && testFlags.CoverProfile != "" {
+		file, _ := json.MarshalIndent(coverageReport, "", "  ")
+		_ = ioutil.WriteFile(testFlags.CoverProfile, file, 0644)
```

Comment:
`ioutil` is deprecated, use `os` instead. Also, it would be good to handle the error instead of silently ignoring it

Reply:
Good catch :100:
Fixed in https://github.com/onflow/flow-cli/pull/827/commits/eec17569d04a2a63063e6bd79eb568f9eccdc2cc